doc/classes/SceneTreeTween.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="SceneTreeTween" inherits="Reference" version="3.5" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Lightweight object used for general-purpose animation via script, using [Tweener]s.
+	</brief_description>
+	<description>
+		Tweens are mostly useful for animations requiring a numerical property to be interpolated over a range of values. The name [i]tween[/i] comes from [i]in-betweening[/i], an animation technique where you specify [i]keyframes[/i] and the computer interpolates the frames that appear between them.
+		[SceneTreeTween] is more suited than [AnimationPlayer] for animations where you don't know the final values in advance. For example, interpolating a dynamically-chosen camera zoom value is best done with a [SceneTreeTween]; it would be difficult to do the same thing with an [AnimationPlayer] node. Tweens are also more light-weight than [AnimationPlayer], so they are very much suited for simple animations or general tasks that don't require visual tweaking provided by the editor. They can be used in a fire-and-forget manner for some logic that normally would be done by code. You can e.g. make something shoot periodically by using a looped [CallbackTweener] with a delay.
```

Comment:
Part of this text is copy-pasted from the original Tween documentation. Instead of repeating it, you could just link to [Tween].
But the part about SceneTreeTween being more light-weight than AnimationPlayer (and normal Tweens) is still relevant.